### DISCUSSION TITLE
wger error on install

### DISCUSSION THREAD

**Discussion by kpirkle86** [wger-project/wger#1083] [General]
I went through several different various installation types to get this on my server. The closest I've gotten is going through portainer.  Page appears to be working but when I start to add exercises it will not populate with known exercises and on the app it will not let me add them at all since it doesn't know what they are. I'm attaching an image of the error I'm getting. I appreciate any help I can get.

Kevin

![image](https://user-images.githubusercontent.com/108526163/176884438-fb7e1961-a3cf-452a-b013-d65e4b1eb09f.png)

**Comment by kpirkle86** [comment]
![Screenshot_20220701-064223](https://user-images.githubusercontent.com/108526163/176894794-7d7b19e9-b832-41e4-a709-3aa7e7ad1364.jpg)

**Comment by rolandgeider** [comment]
Hi! Sorry, I seemed to have missed this. Are you still getting that error on the app? (the other one with the template is ok, it can be ignored)

**Comment by kpirkle86** [comment]
Yeah I kinda gave up on trying to make it work on my own server. Since it
kept failing. You wouldn't by chance have a ISO for it built on just a
raspberrypi image? That would be the quickest way to get it out there and
minimize issue like I had.

**Reply by rolandgeider** [reply]
No, sadly I have no iso image, the closest would be the docker compose setup. Alternatively you could try to install the application "natively", but it's obviously [a bit more work](https://wger.readthedocs.io/en/latest/production.html)
